### ISSUE TITLE
Open Source Friday - OpenFn - January 31, 2025

### ISSUE THREAD

**Issue by stuartc** [githubevents/open-source-friday#113]
### Name

Stu Corbishley

### GitHub Handle

@stuartc

### Tell us about yourself

I'm the CTO and co-founder of OpenFn, an open source workflow automation product used by governments, NGOs and social enterprises.

I've been a developer for about 20 years. I'm passionate about building software that glues things together, and using tools that balance ubiquity and pragmatism. I spent many years primarily using Ruby, and got into Elixir in 2016 - and sprinkling in plenty of Javascript over the years.

I take great satisfaction in keeping solutions lightweight and efficient. Many tools in our sector are unnecessarily large, often involving multi-container and multi-database setups, which I see as a barrier for both new developers and users. 

Elixir has been a fantastic choice in this regard, streamlining the number of components we need to maintain and simplifying the deployment process for those using our software.

### Project Name

OpenFn Lightning

### Project Repo Link

[github.com/OpenFn/lightning](https://github.com/OpenFn/lightning)

### Stream Date

- [X] Yes
- [ ] Not yet

### Dates

01-31-2025

### Twitter URL

@stucorbishley

### LinkedIn URL

[linkedin.com/in/stuartcorbishley](https://www.linkedin.com/in/stuartcorbishley/)

### Additional Information

OpenFn went open-source in 2022 with a complete rewrite of our original application. We're using Elixir & Phoenix LiveView (with a sprinkle of React in places that make sense), Postgres for the db and NodeJS for executing workflows.

Our product sits in a space between incredibly powerful but often expensive and/or complex solutions and bespoke DIY solutions. 

https://www.openfn.org

**Comment by LadyKerr** [comment]
Hey @stuartc are you still available for the stream this week?

**Comment by stuartc** [comment]
> Hey [@stuartc](https://github.com/stuartc) are you still available for the stream this week?
> 
> Apologies for delay in response, we've been OOO

Yep, still am. You've got my email now, so let's sync up there. 😊

**Comment by LadyKerr** [comment]
![Image](https://github.com/user-attachments/assets/afc04937-40f3-4be1-81ae-ec55590a9b2b)
